coops-ui/src/components/Common/SummaryOfficeAddresses.vue

```diff
@@ -0,0 +1,278 @@
+          <div class="meta-container">
+            <label></label>
+            <div class="meta-container__inner"
+              v-if="!isSame(recordsAddress.deliveryAddress, recordsAddress.mailingAddress)"
+            >
+              <label>Mailing Address</label>
+              <div class="address-wrapper">
+                <mailing-address
+                  :address="recordsAddress.mailingAddress"
+                />
+              </div>
+            </div>
+            <span v-else>
+              Mailing Address same as above
+            </span>
+          </div>
+        </li>
+      </div>
+      <div v-else>
+        <li class="container">
+          <div class="meta-container">
+            <label>Records Office</label>
+            <div class="meta-container__inner">
+              <span id="sameAsRegistered">
+                Same as Registered Office
+              </span>
+            </div>
+          </div>
+        </li>
+      </div>
+    </ul>
+  </v-card>
+</template>
+
+<script lang="ts">
+// Libraries
+import { Component, Prop, Mixins } from 'vue-property-decorator'
+
+// Components
+import BaseAddress from 'sbc-common-components/src/components/BaseAddress.vue'
+
+// Mixins
+import { CommonMixin, EntityFilterMixin } from '@/mixins'
+
+// Interfaces
+import { BaseAddressObjIF } from '@/interfaces'
+
+// Enums
+import { EntityTypes } from '@/enums'
+
+@Component({
+  components: {
+    'delivery-address': BaseAddress,
+    'mailing-address': BaseAddress
+  }
+})
+export default class SummaryOfficeAddresses extends Mixins(CommonMixin, EntityFilterMixin) {
+  /**
+   * Registered Office address object passed in from the parent which is pulled from store.
+   */
+  @Prop({ default: null })
+  private registeredAddress: BaseAddressObjIF
+
+  /**
+   * Records Office address object passed in from the parent which is pulled from store.
+   */
+  @Prop({ default: null })
+  private recordsAddress: BaseAddressObjIF
+
+  // Entity Enum
+  private EntityTypes: {} = EntityTypes
+}
+</script>
+
+<style lang="scss" scoped>
+  @import '../../assets/styles/theme.scss';
+  .v-btn {
+    margin: 0;
+    min-width: 6rem;
+    text-transform: none;
+  }
+
+  .reset-btn {
+    margin-top: 0.5rem;
+  }
+
+  .meta-container {
+    display: flex;
+    flex-flow: column nowrap;
+    position: relative;
+  }
+
+  .meta-container__inner {
+    margin-top: 1rem;
+  }
+
+  label:first-child {
+    font-weight: 700;
+    &__inner {
+      flex: 1 1 auto;
+    }
+  }
+
+  @media (min-width: 768px) {
+    .meta-container {
+      flex-flow: row nowrap;
+
+      label:first-child {
+        flex: 0 0 auto;
+        padding-right: 4rem;
+        width: 15.25rem;
+      }
+    }
+
+    .meta-container__inner {
+      margin-top: 0;
+    }
+  }
+
+  .address-list .form {
+    margin-top: 1rem;
+  }
+
+  @media (min-width: 768px) {
+    .address-list .form {
+      margin-top: 0rem
+    }
+  }
+
+  // Address Block Layout
+  .address-wrapper{
```

Comment:
Please add a space before { .

(I wish lint would catch this. Any idea how to configure that?)